lte/gateway/c/session_manager/SessionStateEnforcer.h

```diff
@@ -87,6 +92,8 @@ class SessionStateEnforcer {
    */
   bool set_upf_node(const std::string& node_id, const std::string& n3_addr);
 
+  bool inc_rtx_counter(const std::unique_ptr<SessionState>& session);

```

Comment:
This looks like a function that belongs better in `SessionState` rather than `SessionStateEnforcer`

Reply:
ooo nvm we need the max number.
Then let's use the doxygen style doc here to be more clear about what this function mutates / returns.
Should look something like this
```
  /**
   * TODO: Fill in
   *
   * @param session
   * @return TODO: fill in
   */
```

Reply:
added the doxygen style doc as you suggested